docs/scos/dev/tutorials-and-howtos/howtos/feature-howtos/data-imports/how-to-manage-merchant-custom-prices-for-products-via-data-import.md

```diff
@@ -0,0 +1,45 @@
+---
+title: 'HowTo: Manage merchant custom prices for products via data import'
+last_updated: Feb 04, 2022
+template: howto-guide-template
+---
+
+{% info_block warningBox "B2B only" %}
+
+You can apply the approaches described in this article only if your project follows the B2B model, as the [merchant custom prices](https://docs.spryker.com/docs/scos/user/features/{{site.version}}/merchant-custom-prices-feature-overview.html) is only the B2B feature. Also, keep in mind that these approaches don't work for the Marketplace product offers.
```

Comment:
About

> Also, keep in mind that these approaches don't work for the Marketplace product offers.

We should also add a `warningBox` to say that it is not compatible with Marketplace at the moment